documentation/src/main/wiki/_Sidebar.md

```diff
@@ -9,36 +9,34 @@
         * [[Advanced Setup|AdvancedSetup]]
     * [[Defining Suite|DefiningSuite]]
         * [[Suite Structure|SuiteStructure]]
-        * [[Best practices|BestPractices]]
+        * [[Best Practices|BestPractices]]
     * [[Running Suite|RunningSuite]]
         * [[Tracking Progress|TrackingProgress]]
     * [[Features|Features]]
         * [[Open|Open]]
         * [[Collectors|Collectors]]
             * [[Accessibility|AccessibilityCollector]]
-            * [[Client Side Performance|ClientSidePerformanceCollector]]
```

Comment:
We don't want to have this in TOC ?

Reply:
Nope. It no longer works. ySlow is no longer supported by Chrome and the last github commit took place 5 years ago.

Reply:
ok :(

Reply:
There is the idea to add audits generated with Lighthouse. A corresponding task has been raised at #368. Another ticket to remove redundant code related to ySlow has been raised at #481.